ogc-example/docs/conf.md

```diff
@@ -170,22 +170,6 @@ color-ramps = {
     ]
 ```
 
-A word of caution about color ramps: statistically derived rendering
-strategies can mislead if the entire layer being colored is markedly
-different from a well known distribution. If, for instance, we have NDVI
-values from a recent wildfire and our layer contains little information
-outside of the damaged area, statistically derived styling will see a
-distribution skewing towards -1 (or 0, depending on how the NDVI is
-calculated) and is therefore likely to use too many color breaks in the
-low end of the NDVI range.
-
-In a cases like the one just described, you're better off using a color map
-that is defined specifically for the range of values capable of being
-represented (e.g. for NDVI -1 is red, 0 is yellow, and 1 is green
-regardless of the distribution of NDVI values I happen to be looking
-at). If only the minimum and maximum values are known, they may
-optionally be specified in color ramp configuration (as above).
-
```

Comment:
No longer relevant after the fix to `ColorRamp` behavior (we don't use quantile breaks any longer)